src/common/i18n/locales/es-ES.json

```diff
@@ -5,21 +5,21 @@
   },
   "nav": {
```

Comment:
Don't edit other language files, only en-US.json. If you want to modify some strings, do them in crowdin.com, ask for invite so I can add you there.

Reply:
Sure

Reply:
I edited some of the keys with their respective language translation to test the localization working

Reply:
> I edited some of the keys with their respective language translation to test the localization working

@ahmedaftab-RT Only do that from crowdin.com website so we get update from that service on separate PR.